build.gradle

```diff
@@ -14,13 +14,17 @@ buildscript {
 	repositories {
 		google()
 		mavenCentral()
+		maven {
+			url "https://plugins.gradle.org/m2/"
```

Comment:
For the sake of completeness (and fyi): this repo is [allowlisted](https://gitlab.com/fdroid/fdroidserver/-/blob/master/fdroidserver/scanner.py#L167) by F-Droid, so it should continue to work as before without any extra changes :+1: